### DISCUSSION TITLE
last.fm imports, but does not showing up on Maloja dashboard

### DISCUSSION THREAD

**Discussion by untenops** [krateng/maloja#321] [Q&A]
This is my first time using Maloja.  I set it up in docker and also set up mulit-scrobbler.  My songs are being tracked now and showing up on Maloja.  I exported my history from last.fm.  I shut down Maloja in docker and then started it in shell trying to follow the importing directions.  Seemed to import everything successfully, 1000 tracks at a time.  Looking in my folder the malojadb.sqlite file grew during the import.  I shut down the shell and restarted my maloja docker compose and nothing from my import is showing up in Maloja.  I'm sure i did something wrong.  But don't know what or how to approach this.  Please help. 

Edit - So in hopes of fixing this, I have tried several times again.  2nd attempt I created a new folder for my docker compose and config directory.  Then instead of starting up the container I first ran it in Shell to import my last.fm info.  Again malojadb.sqlite grew in size and all 175,000 scrobbs imported.  Shutdown the shell and started the container and once again no history.
Tried a 3rd time with a new empty config folder.  This time started the container and then shut it down to run it in shell and import my history.  Same results, no history shows up in Maloja.  
I'm sure I'm doing something wrong, but no idea what it could be.

**Comment by Diegos962** [comment]
Actually had the same problem. Solution is simpler than I thought, just "If you would like to import your previous scrobbles, copy them into the import folder in your data directory."
